src/translations/fr/subscribe/done.mdx

```diff
@@ -0,0 +1,5 @@
+# Vous êtes maintenant inscrit !
+
+**Nous vous enverrons les prochaines mises à jour dès qu'elles arriveront.**
+
+Vos commentaires sont toujours les bienvenus, n'hésitez à nous dire comment nous pourrions les e-mails 🙂
```

Comment:
Quite a few words missing in this sentence 😉 

```suggestion
Vos commentaires sont toujours les bienvenus, n'hésitez pas à nous dire comment nous pourrions améliorer les mémos 🙂
```

But let's rephrase in a more precise way:

```suggestion
Nous avons besoins de vos retours : répondez à n’importe lequel de nos mémos pour nous dire comment nous pourrions les améliorer 🙂
```